zerver/migrations/0331_alter_realm_org_type.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.4 on 2021-07-06 20:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0330_linkifier_pattern_validator"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="realm",
+            name="org_type",
+            field=models.PositiveSmallIntegerField(
+                choices=[
+                    (0, "Unspecified"),
+                    (1, "Business"),
+                    (2, "Community"),
+                    (3, "Open-source project"),
+                    (4, "Research"),
+                    (5, "Event or conference"),
+                    (6, "Non-profit"),
+                    (7, "Personal"),
+                    (8, "Other"),
+                    (9, "Political group"),
+                    (10, "Education"),
+                    (11, "Government"),
+                ],
+                default=0,
```

Comment:
This default only affects new Realms; existing ones will incorrectly have the legacy value of 1.  I fixed this by adding a `RunPython` command to the migration.